cmd/tether/tether.go

```diff
@@ -156,12 +156,16 @@ func run(src extraconfig.DataSource, sink extraconfig.DataSink) error {
 
 			// check if session has never been started
 			if proc == nil {
+				// save the modified config back to the vmx file
+				defer extraconfig.Encode(sink, config)
```

Comment:
I'm not sure this will function as desired. I think this is serializing the Started key only when the primary process exits, thereby causing the tether to exit.

Look at line 204 for exit status seralization in the handleSessionExit function - we should be able to do something similar in the launch function for Started.